SignupDirector/SignupDirectorAPI/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import Participants
+
+class ParticipantsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Participants
+        fields = ['id','name','attendance', 'accompanions','event_id']
```

Comment:
Looks like trivial serializer (db field names are the same as class field names).
Framework should allow using some default serializer without explicit class for it to make maintainability better.

Reply:
I propose to leave this issue for later as I don't complete understand what kind of serializer would be better in this case. Lets discuss separately